src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -174,5 +174,11 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <FilesToSign Include="$(OutDir)\$(AssemblyName)$(TargetExt)">
+      <Authenticode>Microsoft</Authenticode>
+      <StrongName>StrongName</StrongName>
+    </FilesToSign>
+  </ItemGroup>
```

Comment:
Yes any changes to a `.csproj` to add a `<FilesToSign />` item can be removed from the merge.